cmd/ak/cmd/projects/download.go

```diff
@@ -14,36 +15,34 @@ import (
 
 var outputDirectory string
 
-var downloadResourcesCmd = common.StandardCommand(&cobra.Command{
-	Use:   "download-resources <project name or ID> [--output-dir=...]",
-	Short: "Download the project's resources",
+var downloadCmd = common.StandardCommand(&cobra.Command{
+	Use:   "download <project name or ID> [--output-dir <path>] [--fail]",
+	Short: "Download project resources",
 	Args:  cobra.ExactArgs(1),
 
 	RunE: func(cmd *cobra.Command, args []string) error {
 		r := resolver.Resolver{Client: common.Client()}
-		_, pid, err := r.ProjectNameOrID(args[0])
+		p, pid, err := r.ProjectNameOrID(args[0])
 		if err != nil {
-			return err
+			return common.FailIfError(cmd, err, "project")
 		}
-
-		if !pid.IsValid() {
-			return fmt.Errorf("project %s not found", args[0])
+		if !p.IsValid() {
+			err = errors.New("project not found")
+			return common.NewExitCodeError(common.NotFoundExitCode, err)
 		}
 
 		ctx, cancel := common.LimitedContext()
 		defer cancel()
 
-		resources, err := projects().DownloadResources(ctx, pid)
+		rs, err := projects().DownloadResources(ctx, pid)
 		if err != nil {
-			return fmt.Errorf("download resources: %w", err)
+			return err
 		}
-
-		if len(resources) == 0 {
-			fmt.Println("no resources found")
-			return nil
+		if err := common.FailIfNotFound(cmd, "resources", len(rs) > 0); err != nil {
+			return err
```

Comment:
why fail?
project can have no resources, it's not an error
also, I'm not sure 
```go
common.FailIfNotFound(cmd, "resources", len(rs) > 0)
```
is more readable than
```go
if len(resources) == 0 {
```

Reply:
@daabr Please open a ticket, we should resolve it somehow
not readable (I know its not on you :) )

Reply:
https://linear.app/autokitteh/issue/ENG-506/clean-up-not-found-error-handling-across-all-cli-commands